pkg/driver/node_test.go

```diff
@@ -1960,6 +1960,7 @@ func TestNodeGetInfo(t *testing.T) {
 				m.EXPECT().GetInstanceID().Return("i-1234567890abcdef0")
 				m.EXPECT().GetAvailabilityZone().Return("us-west-2a")
 				m.EXPECT().GetOutpostArn().Return(arn.ARN{})
+				m.EXPECT().UpdateMetadata().Return(nil)
```

Comment:
Should we maybe add a test for if this returns not nil?

Reply:
Done.